### PR TITLE
Run installed Galaxy with no config and a simplified entry point

### PR DESCRIPTION
**Pull request by natefoo**
Building off https://github.com/galaxyproject/galaxy/pull/17102, make it possible to start and run a simple Galaxy web server using a single command when running from installed packages and the prebuilt client. Also ~~delays conda auto-init util the first time an install is attempted.~~ disables conda auto-init when running installed.

## How to test the changes?
(Select all options that apply)
- [x] Instructions for manual testing are as follows:
  ```sh-session
  python -m venv /tmp/galaxy/venv
  . /tmp/galaxy/venv/bin/activate
  cd packages
  ./package-dev-install.sh web_apps
  cd /tmp/galaxy
  npm install @galaxyproject/galaxy-client
  galaxy-web
  ```

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

**Comment by natefoo** [comment]
This is ready for review

**Review by jdavcs — APPROVED** [review]
Looks good. Although I didn't test the install locally.

File: lib/galaxy/config/schemas/config_schema.yml [hunk 1]
```diff
@@ -386,7 +386,8 @@ mapping:
         required: false
```
Comment (nsoranzo) [inline lib/galaxy/config/schemas/config_schema.yml:385]: I think we should remove the default from the line above like we do for `managed_config_dir` and `data_dir`.

File: lib/galaxy/config/__init__.py [hunk 2]
```diff
@@ -982,6 +982,9 @@ def _process_config(self, kwargs: Dict[str, Any]) -> None:
             # self, populate config_dict
             self.config_dict["conda_mapping_files"] = [self.local_conda_mapping_file, _default_mapping]
 
+        if not running_from_source and kwargs.get("conda_auto_init") is None:
+            self.config_dict["conda_auto_init"] = False
```
Comment (nsoranzo) [inline lib/galaxy/config/__init__.py]:
```suggestion
        if kwargs.get("conda_auto_init") is None:
            self.config_dict["conda_auto_init"] = running_from_source
```

**Review by nsoranzo — APPROVED** [review]

File: lib/galaxy/webapps/base/webapp.py [hunk 3]
```diff
@@ -1156,19 +1156,34 @@ def build_url_map(app, global_conf, **local_conf):
...
-    urlmap["/favicon.ico"] = get_static_from_config("static_favicon_dir", "static/favicon.ico")
-    urlmap["/robots.txt"] = get_static_from_config("static_robots_txt", "static/robots.txt")
+    static_dir = get_static_from_config("static_dir", "static/")
+    static_dir_bare = static_dir.directory.rstrip("/")
+    urlmap["/static"] = static_dir
+    urlmap["/images"] = get_static_from_config("static_images_dir", f"{static_dir_bare}/images")
+    urlmap["/static/scripts"] = get_static_from_config("static_scripts_dir", f"{static_dir_bare}/scripts/")
+
+    urlmap["/static/welcome.html"] = get_static_from_config(
+        "static_welcome_html", f"{static_dir_bare}/welcome.html", sample=default_url_path("static/welcome.sample.html")
```
Comment (bgruening) [inline lib/galaxy/webapps/base/webapp.py:1179]: Shouldn't this be?

```suggestion
        "static_welcome_html", f"{static_dir_bare}/welcome.html", sample=default_url_path("static/welcome.html.sample")
```

**Review by bgruening — APPROVED** [review]

**Pull request merged**